<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiEdit</class>
 <widget class="QWidget" name="MultiEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="btn_add">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/plus_minus/plus.png</normaloff>:/plus_minus/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_rem">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/plus_minus/minus.png</normaloff>:/plus_minus/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../lib2/icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
